src/other/pages/guides/getting-started/story.jsx

```diff
@@ -196,22 +200,22 @@ export default AXAButtonReact;`}
         </header>
         <div className="getting-started__last-text">
           <AXAText variant="size-2">
-            It is our aim to offer our customers security and trust when using
-            our services. To achieve this, we rely on uniform visual
-            communication and use existing and tested interactions.
+            It is our aim to offer our customers safety and trust when using our
+            services. To achieve this, we rely on uniform visual communication
+            and use existing and tested interactions.
           </AXAText>
         </div>
         {callout(
           'figma.png',
           'AXA Design System UI Kit',
-          'To see how to use our AXA Design System and be on sync with the Pattern Library take a look at the AXA UI Kit',
+          "To see how to use AXA's Design System and be in sync with the Pattern Library, take a look at the AXA UI Kit!",
           'https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=0%3A8208',
-          'take a look'
+          'READ More'
```

Comment:
Above you used `READ MORE`. In my opinion I would choose `READ MORE` for all links.

Reply:
Done